### PR TITLE
Update RBAC verbs to include create

### PR DESCRIPTION
**Pull request by geethub97**
## Problem
Error related to services/proxy when viewing metrics in Rancher UI
## Solution
Add create verb to the services/proxy resource RBAC section 

## Testing
Repro steps in the GH ticket linked above are valid.

### CODE REVIEW

**Review by joshmeranda — APPROVED** [review]

**Comment by prachidamle** [comment]
@aiyengar2 could you please review this RBAC change?

File: packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml [hunk 1]
```diff
@@ -131,4 +132,5 @@ rules:
   - endpoints
   verbs:
   - list
+  - 'create'
```
Comment (aiyengar2) [inline packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml]: why do you need endpoints create permissions?

File: packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml [hunk 2]
```diff
@@ -119,6 +119,7 @@ rules:
 {{- end }}
   verbs:
   - 'get'
+  - 'create'
```
Comment (aiyengar2) [inline packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml]: nit: not sure why some of the stuff in this file uses single quotes and some use no quotes, everything should be standardized (probably no quotes)

**Review by aiyengar2 — CHANGES_REQUESTED** [review]

File: packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml (hunk 1 above)
Comment (geethub97) [reply packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/clusterrole.yaml]: This was a miss on my part -- have removed this.

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**